CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Zilliqa Coding and Review Guidelines
+
+1. All submitted **Pull Requests** are assumed to be ready for review.  
+   If not, they must be marked with explicit  `[in progress]` or have "[WIP]" in title.
+
+2. - **Pull requests** which fix a bug should have branch name starting with `fix/`
+   - **Pull requests** which add a feature to the codebase should have branch name starting with `feature/`
+   - **Pull requests** which are made by outside contributors should also contain their name.
```

Comment:
:bulb: pull requests sent by outside contributors will come from a fork, so the user ID of the fork will always appear.

Reply:
yes, sometimes we would give access to contributors to make a branch, in that case.